### PR TITLE
perf(scrollbar): when scrollbar isn't shown, hide the bar

### PR DESCRIPTION
**Pull request by kooriookami**
Please make sure these boxes are checked before submitting your PR, thank you!

* [ ] Make sure you follow Element's contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
* [ ] Make sure you are merging your commits to `dev` branch.
* [ ] Add some descriptions and refer to relative issues for your PR.

### CODE REVIEW

**Comment by Ryan2128** [comment]
Not familiar with this component.💀

**Comment by kooriookami** [comment]
> Not familiar with this component.💀

Haha, you can just look the effect of srcollbar.

File: packages/theme-chalk/src/scrollbar.scss [hunk 1]
```diff
@@ -70,3 +59,16 @@
...
   }
 }
+
+.el-scrollbar-fade-enter-active {
+  transition: opacity 340ms ease-out;
+}
+
+.el-scrollbar-fade-leave-active {
+  transition: opacity 120ms ease-out;
+}
```
Comment (JeremyWuuuuu) [inline packages/theme-chalk/src/scrollbar.scss:69]: Where did you get the transition duration time?

Comment (kooriookami) [reply packages/theme-chalk/src/scrollbar.scss:69]: image
The original sass

File: packages/scrollbar/src/bar.vue [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+
+      wrap.value[bar.value.scroll] = (thumbPositionPercentage * wrap.value[bar.value.scrollSize] / 100)
+    }
+
+    const startDrag = e => {
+      e.stopImmediatePropagation()
+      cursorDown.value = true
+      on(document, 'mousemove', mouseMoveDocumentHandler)
+      on(document, 'mouseup', mouseUpDocumentHandler)
+      document.onselectstart = () => false
```
Comment (JeremyWuuuuu) [inline packages/scrollbar/src/bar.vue:64]: Shouldn't we store the original `onselectstart` handle and restore it after `mouseup`? We can hijack the event, but we can't just set it to null, what do you think?

Comment (kooriookami) [reply packages/scrollbar/src/bar.vue:64]: sounds good

Comment (JeremyWuuuuu) [reply packages/scrollbar/src/bar.vue:66]: You don't need a ref to store this. use a local variable instead.

**Review by jw-foss — APPROVED** [review]

**Pull request merged**